doc/source/index.rst

```diff
@@ -15,3 +15,6 @@ free and open-source solar data analysis environment for Python.
  
 .. warning::
     Having issues with links on this page? Make sure there is a trailing slash on the URL *e.g.* `http://docs.sunpy.org/en/latest/`
+
+-Link to SunPy
+    Want to visit the blog and stay updated with the tutorials?Make sure there is a trailing slash on the URL *e.g.* `http://sunpy.org/`
```

Comment:
What's the problem with the trailing slash? it works for me without it.
Also, Could you add an space after the question mark?

Thanks for contributing :)